### DISCUSSION TITLE
`Actor.GetString()` fails for long strings

### DISCUSSION THREAD

**Discussion by Wesxdz** [nxrighthere/UnrealCLR#225] [Help]
Fails to copy string to CLR correctly at a length of ~21 words

**Comment by nxrighthere** [comment]
I can increase the internal buffer size. Out of curiosity, how many characters do you have there?

**Comment by Wesxdz** [comment]
About 150-200 characters, probably wouldn't expect to go over 256 characters unless I needed to access dialogue, quotes, poems, short stories, etc (maybe that would be better supported by writing out a file though?). It would be helpful if the Actor.GetString and Actor.GetText both supported arbitrary string property lengths.

**Comment by nxrighthere** [comment]
`Actor.GetString()` should be fine in this case. Could you please share the code and the string itself?

**Comment by Wesxdz** [comment]
String is not known until runtime but is a 25 word mnemonic
Currently the mnemonic is split into two properties because it was failing when the string properties were longer
https://github.com/Wesxdz/unreal-algorand/blob/b817ec3a8e6b3ef4faa1e63d914ec03bc928374e/Source/DotNet/SDK/Game.cs#L31
I am calling this code from https://github.com/Wesxdz/unreal-algorand/blob/b817ec3a8e6b3ef4faa1e63d914ec03bc928374e/Source/Algorand/Private/AlgorandBPLibrary.cpp#L21
I am running this code in Unreal Engine Editor Version: 4.27.1-0 on Kubuntu 20.04 LTS

**Comment by nxrighthere** [comment]
Is `Actor.GetString()` returns true in this case? The internal buffer size is 8192 bytes, so it should be enough. The problem might be with the content of the string, but to check that, I need data.

**Reply by nxrighthere** [reply]
Hey @Wesxdz. Any update on this?
